### ISSUE TITLE
Enable Accounts Removal.

### ISSUE THREAD

**Issue by mudlabs** [thepracticaldev/dev.to#254]
### Definition of Done
When a user can disconnect their _Twitter_ or _GitHub_ profile from their _Dev.to_ profile.

### Notes
Currently _Dev.to_ offers no way out. Users should be able to disconnect external profiles form their _Dev.to_ profile.

For example, a user creates their account via _Twitter_, and now wants it connected to _GitHub_. But they don't want the two platforms connected through _Dev.to_. Also users should have this level of control without needing a reason at all.

Further to this would be the ability for users to delete their account altogether.

**Comment by Stargator** [comment]
I would hope to include being able to delete or remove my dev.to account as well.

**Comment by slaFFik** [comment]
@mudlabs You can disconnect Dev.to from Github by going to Github Settings (https://github.com/settings/applications) and clicking `Revoke` for dev.to. 
Same for Twitter here: https://twitter.com/settings/applications

**Comment by mudlabs** [comment]
@slaFFik true. But It would also be good to have this available from the Dev.to profile settings.
